tests/function/test_config.py

```diff
@@ -346,7 +346,7 @@ def test_configs_in_virtwho_conf(self, virtwho, globalconf, hypervisor_data, ssh
 
     @pytest.mark.tier1
     def test_owner_in_virtwho_conf(self, virtwho, globalconf, hypervisor, hypervisor_data,
```

Comment:
the `hypervisor` also can be deleted, but it doesn't matter

Reply:
And https://github.com/VirtwhoQE/virtwho-test/blob/69c3eb25e87b8f07e414b212d550b79dbc7ccf86/tests/function/test_config.py#L389 is the same